app/front/scripts/services/api.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Created by Ihor Borysyuk on 26.01.16.
+ */
+
+;(function (angular) {
+
+  var app = angular.module('Application');
+  app.factory('ApiService', ['$q', '_', 'downloader', function ($q, _, downloader) {
+    var url_api_host = 'http://s145.okserver.org/api/3';
```

Comment:
@akariv If we want to use some  environment variable we should have `backend`. Now we have it but as i know, Viewer should only frontend app.